tasks/updateconstants.js

```diff
@@ -493,6 +493,7 @@ async function start() {
               );
             }
 
+            ability.is_innate = scripts[key].Innate === "1";
```

Comment:
what if we only add this field to innate abilities to save space?

Reply:
sure! fixed